example/build.gradle

```diff
@@ -0,0 +1,46 @@
+apply plugin: 'com.android.library'
```

Comment:
Well, I'll remove useless travis config lines, and make the `example` module a java library.